### PR TITLE
added IMU temperature calibration documentation

### PR DESCRIPTION
**Pull request by tridge**
the corresponding PR should be merged soon:
https://github.com/ArduPilot/ardupilot/pull/16262

### CODE REVIEW

File: common/source/docs/common-imutempcal.rst [hunk 1]
```diff
@@ -0,0 +1,219 @@
+.. _common-magfit:
```
Comment (Hwurzburg) [inline common/source/docs/common-imutempcal.rst:1]: needs to match the file name

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by Hwurzburg** [comment]
suggest a title Online Temperature Calibration since you have an Offline one...

**Comment by Hwurzburg** [comment]
builds okay but params are unlinked since the code has not merged.....will need to add this to the Upcoming features page when we are ready to merge to get the TOC entry and it belongs there since not in a release yet....

File: common/source/docs/common-imutempcal.rst (hunk 1 above)
Comment (tridge) [reply common/source/docs/common-imutempcal.rst:1]: fixed

**Comment by tridge** [comment]
> suggest a title Online Temperature Calibration since you have an Offline one...

the doc covers both

**Comment by tridge** [comment]
> builds okay but params are unlinked since the code has not merged.....will need to add this to the Upcoming features page when we are ready to merge to get the TOC entry and it belongs there since not in a release yet....

yep, I didn't link it at all yet. We can do that when ardupilot PR merged

**Comment by Hwurzburg** [comment]
> > suggest a title Online Temperature Calibration since you have an Offline one...
> 
> the doc covers both

so you can start right at the section titled Offline.... without the predecessor? may be just editorial fancy, but normally I would expect a flow like (not to suggest titles, just content partitioning): 

-common setup
-Online cal
-Offline cal
-Factory cal

we start by stating two methods.... by only one is subtitled...

**Comment by tridge** [comment]
> we start by stating two methods.... but only one is subtitled...

that is because the process for doing an offline cal starts by doing an online cal. Happy to change it if you think it will work better of course

**Comment by tridge** [comment]
PR is now merged, so we can add this into the TOC

**Comment by Hwurzburg** [comment]
merged vai another branch

**Pull request closed without merging**